tests/unit/test_cli_install_status.py

```
@@ -112,3 +116,35 @@ def test_doctor_includes_cli_install(monkeypatch: pytest.MonkeyPatch) -> None:
+        }
+    )
+    assert "via=uv_tool" in human
+    assert "make cli-status" in human
+    assert "make cli-install" in human
+
+
+def test_collect_cli_install_status_omits_hint_without_uv_tool(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    monkeypatch.setattr(cis.shutil, "which", lambda _name: None)
+    monkeypatch.setattr(cis, "_potpie_paths_on_path", lambda: [])
```

Ruff (0.15.21)

[warning] Prefer `list` over useless lambda

Replace with `lambda` with `list`

(PIE807)